src/rgw/rgw_gc.cc

```diff
@@ -2,13 +2,16 @@
 // vim: ts=8 sw=2 smarttab
 
 #include "rgw_gc.h"
+
+#include "include/scope_guard.h"
 #include "include/rados/librados.hpp"
 #include "cls/rgw/cls_rgw_client.h"
 #include "cls/refcount/cls_refcount_client.h"
+#include "rgw_perf_counters.h"
```

Comment:
This line causes build failure:

```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/rgw/rgw_gc.cc:10:10: fatal error: rgw_perf_counters.h: No such file or directory
 #include "rgw_perf_counters.h"
          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
src/rgw/CMakeFiles/rgw_a.dir/build.make:313: recipe for target 'src/rgw/CMakeFiles/rgw_a.dir/rgw_gc.cc.o' failed
make[3]: *** [src/rgw/CMakeFiles/rgw_a.dir/rgw_gc.cc.o] Error 1
```

I guess this line can be dropped, since we don't have this file in mimic.

Reply:
:+1: